packages/lesswrong/components/posts/AllPostsPage.jsx

```diff
@@ -102,16 +105,22 @@ class AllPostsPage extends Component {
     }
 
     return <div className={classes.timeframe}>
-      <PostsTimeframeList
-        timeframe={currentTimeframe}
-        postListParameters={postListParameters}
-        numTimeBlocks={numTimeBlocks}
-        dimWhenLoading={showSettings}
-        after={query.after || getAfterDefault({numTimeBlocks, timeBlock, timezone})}
-        before={query.before  || getBeforeDefault({timeBlock, timezone})}
-        reverse={query.reverse === "true"}
-        displayShortform={query.includeShortform !== "false"}
-      />
+      <AnalyticsContext
+          listContext={"allPostsPage"}
```

Comment:
Indentation nitpick: the preferred style when you have props that you want to split onto multiple lines looks like

```
<AnalyticsContext
  listContext="allPostsPage"
>
```

with the closing `>` on its own line, and the props indented one level.